src/common/restext.cpp

```diff
@@ -626,7 +628,7 @@ void InitializeRestext()
     stringsErr[ERR_BASE_DLOCK]      = TR("Doors blocked by a robot or another object");
     stringsErr[ERR_BASE_DHUMAN]     = TR("You must get on the spaceship to take off");
     stringsErr[ERR_LABO_NULL]       = TR("Nothing to analyze");
-    stringsErr[ERR_LABO_BAD]        = TR("Analyzes only organic matter");
+    stringsErr[ERR_LABO_BAD]        = TR("Inappropriate sample");
```

Comment:
Just looking through old pull requests, you didn't run `make update-pot` after changing restext.cpp and I still didn't add a comment in this file explaining that you have to to that :P

Reply:
It's now done so I'm resolving this.